### PR TITLE
Updated privacy policy to account for events.

### PR DESCRIPTION
**Pull request by chriseyre2000**
This is an update of the privacy policy to add events.

This will allow us to share this across codurance.com and sc-london.com

We can't go live with this until privacy@codurance.com exists.

### CODE REVIEW

**Comment by chriseyre2000** [comment]
Here is the updated reference document: https://docs.google.com/document/d/1z5hXhcTJ7bu86LX5PILR0qvmyyQPwj_Hb0KPLVGDbQ8/edit

It is now safe to merge as we now have a privacy@codurance.com email address.

File: src/policies/privacy.html [hunk 1]
```diff
@@ -46,41 +46,34 @@ <h4>How will Codurance use this information?</h4>
                 <li>To communicate with you regarding your query or application. We may contact you by post, email, telephone or text message. You may opt out/unsubscribe from such communications at any time by emailing privacy@codurance.com;</li>
                 <li>To process any job application you may make to us (including any sensitive personal information you provide);</li>
                 <li>To provide or administer services;</li>
+                <li>To communicate, provide and administer events;</li>
                 <li>To update client records;</li>
                 <li>To confirm identity, ascertain credit worthiness and trace whereabouts where necessary;</li>
                 <li>To comply with legal and regulatory requirements including verifying the identity of new clients/associates (and in certain circumstances existing clients/associates) to comply with anti-money laundering regulations, ‘right to work’ and other legislative requirements applicable to Codurance;</li>
                 <li>To comply with law enforcement.</li>
+                <li>To contact you about a submisison you have mades to the websites, including any content you provide;</li>
```
Comment (alfredodev) [inline src/policies/privacy.html:54]: Typo? in you have mades

File: src/policies/privacy.html [hunk 2]
```diff
@@ -46,41 +46,34 @@ <h4>How will Codurance use this information?</h4>
...
-            <p>If your personal information changes or you no longer wish to receive a particular communication from us please let us know and we will correct, update or remove your details. This can be done by emailing hello@codurance.com</p>
+            <p>If your personal information changes or you no longer wish to receive a particular communication from us please let us know and we will correct, update or remove your details. This can be done by emailing privacy@codurance.com</p>
         </div>
         <div class="margin-bottom-30">
             <h4>Your agreement</h4>
-            <p>By entering your details in the fields requested, you enable Codurance and its service providers to provide you with the services you select.</p>
+            <p>By entering your details in the fields requested, you enable Codurance and its service providers to provide you with the services you selected and/or events you have registered for.</p>
```
Comment (alfredodev) [inline src/policies/privacy.html:65]: All the bullets ends with a semicolon except for this one

**Comment by alfredodev** [comment]
Should we change the email to privacy@codurance.com also in the cookies policy?

**Pull request merged**